MAMConverter/SelectTimeZoneViewController.h

```diff
@@ -0,0 +1,25 @@
+//
+//  SelectTimeZoneViewController.h
+//  MAMConverter
+//
+//  Created by Abu Khalid on 2/10/16.
+//  Copyright © 2016 Md Mainul Haque. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@protocol SelectTimeZoneViewControllerDelegate <NSObject>
+-(void) selectTimeZone:(NSString *)nameTimeZone;
+@end
+
+
+@interface SelectTimeZoneViewController : UIViewController <UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate>
+
+@property (nonatomic, weak) IBOutlet UITableView *selectTimeZoneTableView;
```

Comment:
According to [Coding Convention](https://github.com/raywenderlich/objective-c-style-guide) 
we can define property like this
`@property (weak, nonatomic) IBOutlet UITableView *selectTimeZoneTableView;`
meaning atomicity later than storage.
We can follow this